packages/fx-core/resource/package.nls.json

```diff
@@ -562,6 +562,7 @@
   "core.summary.createdEnvFile": "Environment file was created at",
   "core.copilot.addAPI.success": "%s have(has) been successfully added to %s",
   "core.copilot.addAPI.InjectAPIKeyActionFailed": "Inject API key action to teamsapp.yaml file failed, please make sure that file contains teamsApp/create action in provision section.",
+  "core.copilot.addAPI.InjectOAuthActionFailed": "Inject OAuth action to teamsapp.yaml file failed, please make sure that file contains teamsApp/create action in provision section.",
```

Comment:
Revised: Inject OAuth action to teamsapp.yaml file unsuccessful, make sure the file contains teamsApp/create action in provision section.

Reply:
Updated